doc/sphinx/source/introduction.rst

```diff
@@ -160,7 +151,24 @@ This makes for excellent cheating in debug builds, but note that this is for
 debug purposes only -- in network games running Lua commands this way will
 desync and therefore crash the game and also replays where you changed the
 game state via the debug console will not work. It is very useful
-for debugging scenarios though.
+for debugging scenarios though. It is also possible to load a script from any
+directory which makes testing of functions very easy. Let's assume you test
+a function like:
+
+.. code-block:: lua
+
+   function all_players()
+      for idx, player in ipairs(wl.Game().players) do
+        print("Player:" player.name, player.number, player.tribe.name) 
+      end
+   end
+
+Save this as ``tests.lua``. Now start a normal game, open the debug console 
+by pressing ``SHIFT+CTRL+SPACE`` and enter ``dofile("/full/path/to/tests.lua")``.
+Now you can run the function ``all_players()``. If the output is not what you
+expected just change the function, load the file again with ``dofile()`` and 
```

Comment:
```suggestion
expected just change the function, load the file again with ``dofile`` and 
```
otherwise people might think this function needs to be called without arguments to reload the file

Reply:
done